app/src/main/kotlin/com/wire/android/ui/sharing/ImportMediaScreen.kt

```diff
@@ -116,19 +127,41 @@ fun ImportMediaScreen(
 
         FeatureFlagState.SharingRestrictedState.NONE -> {
             val importMediaViewModel: ImportMediaAuthenticatedViewModel = hiltViewModel()
+            val sendMessageViewModel: SendMessageViewModel = hiltViewModel()
+
+            LaunchedEffect(sendMessageViewModel.viewState.afterMessageSendAction) {
+                when (val action = sendMessageViewModel.viewState.afterMessageSendAction) {
+                    SendMessageAction.NavigateBack -> navigator.navigateBack()
+                    is SendMessageAction.NavigateToConversation -> navigator.navigate(
+                        NavigationCommand(
+                            ConversationScreenDestination(action.conversationId),
+                            BackStackMode.REMOVE_CURRENT
+                        )
+                    )
+
+                    SendMessageAction.NavigateToHome -> navigator.navigate(
+                        NavigationCommand(
+                            HomeScreenDestination(),
+                            BackStackMode.REMOVE_CURRENT
+                        )
+                    )
```

Comment:
same question here on making new `Home` Screen to be the start on the stack

Reply:
I will do refactor to avoid that function here